types/index.d.ts

```diff
@@ -92,6 +92,14 @@ used to order imports within each match group.
      * @default ["typescript", "jsx"]
      */
     importOrderParserPlugins?: ImportOrderParserPlugin[];
+
+    /**
+     * A choice value to enable sorting imports within their groups based on their string lengths, the two options being ascending and descending.
+     * Leaving the value blank or setting it to null will result in length being ignored
+     *
+     * @default undefined
```

Comment:
Should this not be null instead of undefined?
```suggestion
     * @default null
 ```